portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,28 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// Show current date and time on every page
+function showDate() {
+    const date = new Date();
+    const dateContainer = document.getElementById("date");
+    dateContainer.style.color = "white";
+    dateContainer.innerHTML = date;
+} 
+
+function displayExperience() {
+    document.getElementById("TravelExp").classList.toggle("show");
+}
+
+window.onclick = function(event) {
+  if (!event.target.matches('.dropbutton')) {
+    var dropdowns = document.getElementsByClassName("dropdown-content");
+    var i;
+    for (i = 0; i < dropdowns.length; i++) {
```

Comment:
unlike c/c++, we should declare the variable inside the loop definition:
`for (let i = 0; i < ...`
which looks a little nicer imo